versioned_docs/version-8.6/apis-tools/java-client/index.md

```diff
@@ -6,7 +6,7 @@ description: "Provide a job worker that handles polling for available jobs, use
 ---
 
 :::info
-Starting with 8.7, the Zeebe Java client will become the new Camunda Java client. This transition brings a new Java client structure designed to enhance the user experience and introduce new features while maintaining compatibility with existing codebases.
+Starting with 8.8, the Zeebe Java client will become the new Camunda Java client. This transition brings a new Java client structure designed to enhance the user experience and introduce new features while maintaining compatibility with existing codebases.
```

Comment:
I think we can remove this from 8.6. This is important for 8.7 users. Do we need to give a 2-versions-ahead note already in 8.6? People cannot do anything about it in 8.6 and when they move to 8.7, they can't either.